wpilibj/src/athena/java/edu/wpi/first/wpilibj/DoubleSolenoid.java

```diff
@@ -34,15 +32,13 @@
     kReverse
   }
 
-  private int m_forwardChannel; // /< The forward channel on the module to control.
-  private int m_reverseChannel; // /< The reverse channel on the module to control.
-  private byte m_forwardMask; // /< The mask for the forward channel.
-  private byte m_reverseMask; // /< The mask for the reverse channel.
+  private byte m_forwardMask; // The mask for the forward channel.
+  private byte m_reverseMask; // The mask for the reverse channel.
```

Comment:
They were broken by commit 7eb8550bdb50e7fe7270ff49f5086159cde3db57, which is the initial reformatting commit from summer 2015.